MagazineLayout/LayoutCore/SectionModel.swift

```diff
@@ -97,13 +102,32 @@ struct SectionModel {
     // so we can't use a copy made before that code executes (for example, in a
     // `guard let headerModel = headerModel else { ... }` at the top of this function).
     if let headerModel = headerModel {
-      return CGRect(origin: headerModel.originInSection, size: headerModel.size)
+      let originY: CGFloat
+      if headerModel.pinToVisibleBounds {
+        originY = max(
+          min(
+            sectionVisibleBounds.minY,
+            calculateHeight() -
+              metrics.sectionInsets.bottom -
+              (footerModel?.size.height ?? 0) -
```

Comment:
Looks like this means you've tested a section with a sticky header and footer?

Reply:
Yes, I didn't make a GIF of it though... it works well with both enabled at the same time 🙂